pages/mesosphere/dcos/1.14/installing/production/deploying-dcos/installation/index.md

```diff
@@ -275,6 +275,15 @@ enable_ipv6: 'false'
     - 'foo.bar.com'
     - '.baz.com'
 
+[enterprise]
+## Create a bootstrap PSK (Optional)
```

Comment:
`PSK` is not defined as a shortcut. Let's use the complete word here and define PSK somewhere in the text.